### DISCUSSION TITLE
Any tips on how to make a DVR effect?

### DISCUSSION THREAD

**Discussion by TheGithubberpng** [ntsc-rs/ntsc-rs#150] [Q&A]
I need help with making a DVR effect.

**Comment by Degamisu** [comment]
What do you mean by DVR? The drone type?

**Reply by Asyahril395** [reply]
@Degamisu This is not a drone type, this Digital Video Recorder (DVR) is a tool for monitoring and recording image objects visible by CCTV cameras 😀

**Reply by Asyahril395** [reply]
Oh, I think I should have added a Vertical Interval Timecode? 🤔 (I'm just asking)

![SVHS-1993-VITC-Timecode-928x624-VHS-Decode](https://github.com/user-attachments/assets/38d5b2e4-f1d2-4c45-93d2-66b3db9b36f9)
![Vitc-PAL](https://github.com/user-attachments/assets/802cb398-5c99-4816-947f-ab952f94d878)

according to the image on Google, "DVR effect" 😊
Sorry, the question is a bit awkward and random 🙂
